Automattic/pocket-casts-ios
# Cleanup: Removes the TermsViewController

Removes TermsViewController and references to it. 

## To test

1. In Xcode, open `MainTabBarController` and add `showPromotionFinishedAcknowledge()` to viewDidAppear on line 79
2. Launch the app
3. The promotion finished view should appear
4. Tap the upgrade button
5. ✅ Verify you are brought to the new upgrade flow

## Checklist
